source/docs/hardware-components/fasteners-guide.rst

```diff
@@ -0,0 +1,182 @@
+
+For major diameters of |frac14|" or greater, first number is simply the major diameter in inches. For those smaller than |frac14|", the following code is used:
+
+.. table::
+   :widths: auto
+
+   =====  =====
+    no.    major diameter (inches)
+   =====  =====
+   #0     0.0600
+   #1     0.0730
+   #2     0.0860
+   #3     0.0990
+   #4     0.1120
+   #5     0.1250
+   #6     0.1380
+   #8     0.1640
+   #10    0.1900
+   #12    0.2160
+   =====  =====
+
+
+Some common imperial thread sizes in FTC include |frac14|-20 (one quarter inch major diameter, 20 threads per inch) and #6-32 (0.138 inch major diameter, 32 threads per inch)
+
+**Metric (ISO)**
+
+The ISO metric thread standard is commonly used worldwide, and for many products sold by FTC vendors.
+
+A metric thread is designated by the letter M followed by the outer diameter in millimeters, for example M4 for a 4 mm major diameter bolt. This is then followed by a "x" and the pitch, also in millimeters. For example, "M4X0.7" means a m4 thread with a 0.7 mm pitch.
+
+Bolts
+^^^^^
+
+Bolts are probably the most commonly used fastener in FTC.
+
+There are several types of bolts you may encounter, such as:
+
+**Socket Head Bolts**
+
+.. figure:: images/socket-head.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These bolts have a cylindrical head and are tightened and loosened with a hexagonal recess.
+
+**Button Head Bolts**
+
+.. figure:: images/botton-head.jpeg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These bolts have a domed head, in contrast to the flat top of a standard socket bolt.
+
+**Grub Screws**
+
+.. figure:: images/grub-screw.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These bolts have no head, the recess is instead embedded within the threads. They are primarily used as set screws.
+
+**Hex Bolts**
+
+.. figure:: images/hex-head.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+
+The bolts are tightened with an external hex-shaped head. When designing for these bolts, it is important to allow clearance on the side of the head for tightening.
+
+
+Every bolt requires a "drive" -- meaning the recess which is used to turn and install it. Each of the bolts listed above can have a variety of different types of drives:
+
+.. figure:: images/types-of-drives.png
+   :alt: diagram showing thread terminology
+
+Nuts
+^^^^
+
+A nut is a fastener with an internally threaded hole. Together with a bolt, they're used to join parts together.
+
+Here are some types of nuts you may encounter in FTC:
+
+
+**Hex Nuts**
+
+.. figure:: images/hex-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These most common type of nut. They can be tightened with a socket hex wrench
+
+**Square Nuts**
+
+.. figure:: images/square-nut.jpg
+   :alt: diagram showing thread terminology
+   :width: 150px
+
+These are four-sided nuts.
```

Comment:
While this is true, there are reasons people use them in specific circumstances that are worth mentioning (mainly around nut pockets, but I could be wrong).